### ISSUE TITLE
Pier start fail

### ISSUE THREAD

**Issue by jzhe886** [meshplus/pier-client-fabric#4]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. What did you do?
Start the pier.

### 3. What did you see instead?
![image](https://user-images.githubusercontent.com/17755623/78569471-45b17600-7856-11ea-90dd-6b6feb1da8e1.png)

### 4. What version of BitXHub are you using? (`bitxhub version` on BitXHub)

**Pier version:**
```shell
Pier version: 1.0.0-cd3ebcf
App build date: 2020-04-06T22:25:14
System version: darwin/amd64
Golang version: go1.13.5
```

**Comment by Alexader** [comment]
This Bug is because the version of "github.com/spf13/pflag" is different in `pier` and `pier-client-fabric`. When `pier` starts to run, pier has to load the `.so` file which is compiled out of `pier-client-fabric`.

**Comment by Alexader** [comment]
The version problem is fixed via #5 .
